### DISCUSSION TITLE
Stuck on "We think you’re gonna like it here"

### DISCUSSION THREAD

**Discussion by craig-shenton** [community/community#45576] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

My organisation home page now says "We think you’re gonna like it here." with a list of tasks however, changing member persissions has no effect. I'm not sure prompting people to change these permissions is a good idea in the first place.

<img width="294" alt="Screenshot 2023-01-27 at 19 58 29" src="https://user-images.githubusercontent.com/68001684/215185652-23335950-f897-4d5e-86a6-0abba3eb1752.png">

Any tips on how to get past this point?

**Comment by jge162** [comment]
What exactly is your question? Are you trying to remove the tab? Or looking for an alternative?

**Reply by jge162** [reply]
you can remove the message from your GitHub organization's homepage. You can do this by editing the home page in the GitHub repository.

**Reply by craig-shenton** [reply]
It seems like a new feature that pops up, not part of the .github README. Not sure if it is automatic or another admin clicked the "Get started with tasks that most successful organizations complete" button. But following the task to change user permissions does not complete the list. You can click the "You can hide the tasks on this page and bring them back later" button to get rid of the list, but the final task is still left not complete.

**Reply by jge162** [reply]
@craig-shenton If the tasks on the page are not completing, it could be due to a technical issue or an issue with the user's permissions. If it is a technical issue, then you should contact [GitHub Support](https://support.github.com/contact/bug-report) for help resolving it. If it is an issue with the user's permissions, then you can look at the user's permissions in the repository settings and make sure they have the proper permissions to complete the tasks.

**Comment by landsman** [comment]
I really hate this wizard. Please add button to get rid of it! 🙏 😠

**Reply by joelzamboni** [reply]
Please!

**Reply by mikkokotila** [reply]
This is only useful for people who create a repo for the first time. It should be not the default behavior for all new repo creations.

**Comment by landsman** [comment]
There is action that let you skip it, but it's totally invisible on the right side, column.
Please move this action under or above this wizard...

**Reply by ell1e** [reply]
Oh, wow, that's almost impossible to find. I hope that'll be moved into a way more prominent spot. Thanks for mentioning where it is!

**Reply by darvidor** [reply]
The action is "You can hide **the task we've suggested**  on this page and bring them back later." It is a link on the right column. Just for clarification.

**Reply by edwardofclt** [reply]
Thanks @darvidor - I would have never found it

**Reply by darvidor** [reply]
@edwardofclt :)

**Reply by dant3** [reply]
You saved my day. I went through that panel quite a few time, but only after reading your message I finally found that skip link among the text! Thanks!

**Reply by austinlasseter** [reply]
thanks, this was a life-saver. my organization is customer facing. this set of suggested tasks is not only useless and ugly but potentially damaging. we don't want to change anybody's permissions for Petes sake.

**Reply by larry0x** [reply]
Thank you so much @darvidor

**Reply by mattmc3** [reply]
Here's a screen capture to help you find it:

<img width="311" alt="Screenshot 2024-09-02 at 12 44 10 PM" src="https://github.com/user-attachments/assets/e26b0f42-abf1-4a6b-b03a-e7fbe060d4aa">

Just ran into this today and couldn't figure it out without Google taking me here - to fast track people looking for how to ditch this, it's the "hide the tasks we've suggested" link pictured. Not sure why I need GitHub to assign me tasks, nor why it's so tricky to skip 🤷

**Reply by nomadshiba** [reply]
i was about to go mad, not being able to see the repos directly on the overview tab.

**Reply by firminochangani** [reply]
@mattmc3 THANK YOU!!!

**Comment by landsman** [comment]
Guys, any update? It's a small easy thing to fix.

**Reply by fvarrui** [reply]
![image](https://github.com/community/community/assets/4354486/3dabab16-6396-44a4-aaa0-4e474816238c)

**Reply by ell1e** [reply]
Yeah it's there, but why is it hidden like that? And IMHO, especially after the project was already populated with pinned projects and has another member, it's kind of weird that it still sticks around as the default screen.

IMHO it gets in the way of just doing relevant stuff, rather than gamified checklist tasks, at some point.

**Reply by landsman** [reply]
@fvarrui it's basically invisible... not ux friendly

**Reply by fvarrui** [reply]
🤷‍♂️

**Reply by Kagami** [reply]
Spent 10 minutes trying to find that button 😄  Only google search to this discussion helped.

**Reply by bmovement** [reply]
Oh my god I hate this, thank you!

**Comment by manishtiwari25** [comment]
There is a link/button to hide this, but it's very hard to find on the right side, column

![image](https://github.com/community/community/assets/50008945/2b08b3c8-6271-4c2f-b24b-12e6cb0b0128)

**Reply by d1y** [reply]
Thank you, I really hate this guide. I feel like I'm being treated like a baby, even though I've been using Github for many years and I'm already very familiar with it

**Reply by SirWrexes** [reply]
Can't believe I had to search for this thread to find it.
I was like "I just pinned repos, why am I still seeing this annoyance ?"
I think it should be automatically hidden when you pin some repo(s) to your organisation, that would probably get rid of some frustrations about this landing page layout—and it just makes sense tbh.

**Reply by eshattow** [reply]
At least 20+ months later we have new common language terminology to discuss the old issue: #microslop #enshittification

**Comment by ghost** [comment]
I found the button, but it took a few minutes. The button should be a lot more obvious, this message is kind of annoying.

**Comment by ooobii** [comment]
Thanks to everyone for showing the link's location 😄 this was driving me mad!

The link that allows you to remove this tutorial should be much-more prominent towards the bottom of the goal / tasks list (maybe a button?), instead of hidden as a small hyperlink that can easily be overlooked.

**Comment by njfamirm** [comment]
I'm really proud of our field
We're really cool people 😂😂

Not many people realize that some problems can be solved

**Comment by dacrystal** [comment]
In 2026, and this threads is still saving lives. Thanks @landsman
